docs/nextflow_schema/nextflow_schema_specification.md

```diff
@@ -102,12 +100,28 @@ or on the CLI:
 nextflow run <pipeline> --foo.bar "baz"
 ```
 
-But - the current implementations of the Nextflow _schema_ do not (for now).
+Nested parameters can be specified in the schema by adding a `properties` keyword to the root parameters:
 
-This was done as a conscious decision when the code was first written, to try to reduce complexity.
+```json
+{
+  "type": "object",
+  "properties": {
+    "nested": {
```

Comment:
Very minor concern, but I wonder if people may misunderstand this example and think that `nested` needs to be the name for `nested` params? 
If it were `thisIsNested` or something it may be clearer this is just a dummy name example?

Reply:
Good idea! :partying_face: